File: IAL-002/Listas/4-Funções/04.py
def produto(a, b):
    # Recebe: um inteiro a (a>=0) e um inteiro b (b>=0).
    # Retorna: um inteiro com o produto de a por b.
    # Obs: nao usar o operador de multiplicacao.
    prod = 0
    for i in range(b):
        prod = prod+a
    return prod

File: ILP-010/Listas/1-Funcoes/04.c
int divide(int p, int q){
//Recebe: dois inteiros p e q (p >= 0  e q > 0).
//Retorna: o quociente da divisao inteira de p por q.
//Obs: nao usar operador de divisao.
  int quo;
  for(quo=0; p>=q; p-=q)
    quo++;
  return quo;
}